modules/local/busco.nf

```diff
@@ -25,17 +24,16 @@ process BUSCO {
 
     script:
     def cp_augustus_config = workflow.profile.toString().indexOf("conda") != -1 ? "N" : "Y"
-    def lineage_dataset_provided = params.busco_reference ? "Y" : "N"
+    def lineage_dataset_provided = "${db_meta.lineage}"
     def busco_clean = params.busco_clean ? "Y" : "N"
 
-    def p = "--auto-lineage"
-    if (params.busco_reference){
+    def p = params.busco_auto_lineage_prok ? "--auto-lineage-prok" : "--auto-lineage"
```

Comment:
The `p` variable and `auto-lineage` parameter are both already in the busco module. I just shortened it to a one liner. Also, if I recall correctly, passing in an empty database to busco with `--auto-lineage`, it will automatically download lineages.

I'm not super familiar with the `ext.args` but looking into it, it does make sense. I'm open to suggestions on handling the lineage datasets (full local db vs lineage dataset).

Reply:
Ah ok... if it was already in there just leave it as it is!

More info on ext.args: https://nf-co.re/docs/contributing/modules#what-are-the-ext-propertieskeys

Although to be honest we should rip out all of these local modules and replace with 'offiical' nf-core ones...